pharus/interface.py

```diff
@@ -413,14 +417,3 @@ def set_datajoint_config(jwt_payload: dict):
         dj.config['database.user'] = jwt_payload['username']
         dj.config['database.password'] = jwt_payload['password']
         dj.conn(reset=True)
-
-    @staticmethod
-    def snake_to_camel_case(string: str):
-        """
-        Helper method for converting snake to camel case
-        :param string: String in snake format to convert to camel case
-        :type string: str
-        :return: String formated in CamelCase notation
-        :rtype: str
-        """
-        return ''.join(string_component.title() for string_component in string.split('_'))
```

Comment:
I originally implmented this function to get more perforance out since the cases that we were dealing with didn't need complicated regex which is used in datajoint.to_camel_case.

Reply:
regex is very efficient, in most cases more so than python string functions.

Reply:
@Synicix While I do agree that there could be some performance boost from doing it as you suggest given that table names are usually short strings, I think it is still worthwhile to use the imported method purely from the perspective of preventing duplication of code and to keep maintenance/review cost low. If the concern is performance with the `datajoint.utils.to_camel_case` the appropriate place would be to address it in DJ so we can reach the highest amount of benefit for the user community. We can easily verify if your previous solution here would be sufficient by making the small change in DJ and rerunning the tests.

Reply:
@guzman-raphael Yeah duplication of code is a problem, but the main purpose here was just to deal with a subset of to_camel_case, not so much of replacing it.